ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -84,6 +84,15 @@ public enum TableViewCellBackgroundStyleType: Int {
     }
 }
 
+// Supported vertical alignment for accessory views in `TableViewCell`
+@objc(MSFTableViewCellAccessoryViewVerticalAlignment)
+public enum TableViewCellAccessoryViewVerticalAlignment: Int {
+    // Accessory view is vertically aligned with the top of its corresponding label
+    case top
+    // Accessory view is vertically centered with its corresponding label
+    case center
```

Comment:
Something we should follow up with design about is if we want to support both top and center, or just top.

Reply:
Looks like we should only support top-align for label accessories!